libs/api-hjarntorget/lib/apiHjarntorget.ts

```diff
@@ -571,6 +572,22 @@ export class ApiHjarntorget extends EventEmitter implements Api {
     return statusChecker
   }
 
+  public async registerAbscense(child: EtjanstChild, startDate: Date, endDate: Date): Promise<void> {
+
+      const body = {
+        attendeeId: child.id,
+        startDate: startDate,
+        endDate: endDate,
+        statusId: 27433608,
+        _submit: 'Register+the+whole+day'
+      }
+
+     this.fetch('register-abscense', abscenseRegistrationUrl, {
```

Comment:
I tried making this call, but I get a `405` response back ?!? This is strange to me since it seems to indicate that this shouldn't be submitted with a POST, but that is what happens in the webapp. 🤔 Maybe the initial GET of `https://hjarntorget.goteborg.se/attendanceParentRegisterAbsence.do?attendeeId=<child_id>` is needed to prep the system? This is a bit weird though...